setup.py

```diff
@@ -35,25 +24,27 @@
             continue
 
 
+install_requires = [
+    'httpx>=0.16',
+    'tqdm>=4.56',
+    'pywin32 >= 1.0;platform_system=="Windows"'
```

Comment:
can we confirm that this works? I have no way of testing Windows stuff unfortunately. If we want to constrain to Python >3.5 support it might be better to do `pywin >= 300` (ref: https://github.com/mhammond/pywin32/releases/tag/b300)

Reply:
Good catch, this is a weak point. First, I'll admit this is left over from v1 and hasn't really gotten any attention in v2. The whole packaging and distribution process is being overhauled. As per our offline discussion, we will remove this dependency for now and address supporting windows in the build/distribution process in #264.